c2rust-analyze/src/context.rs

```diff
@@ -84,11 +85,25 @@ pub struct AnalysisCtxt<'a, 'tcx> {
 
     pub local_decls: &'a LocalDecls<'tcx>,
     pub local_tys: IndexVec<Local, LTy<'tcx>>,
+    pub c_void_ptrs: IndexSet<Place<'tcx>>,
     pub addr_of_local: IndexVec<Local, PointerId>,
     /// Types for certain [`Rvalue`]s.  Some `Rvalue`s introduce fresh [`PointerId`]s; to keep
     /// those `PointerId`s consistent, the `Rvalue`'s type must be stored rather than recomputed on
     /// the fly.
     pub rvalue_tys: HashMap<Location, LTy<'tcx>>,
+    /// A mapping for substituting [`Place`]s adhering to the
+    /// following pattern
+    /// ```
+    /// _1 = malloc(...);
+    /// _2 = _1 as *mut T;
+    /// ```
```

Comment:
```suggestion
    /// ```mir
    /// _1 = malloc(...);
    /// _2 = _1 as *mut T;
    /// ```
```

Otherwise I think it may be interpreted as a Rust doctest, and also it's more clear.  This also applies to the other ` ``` `s.

Reply:
addressed in 0d6dc10f